ui/css/treeherder-pinboard.css

```diff
@@ -136,6 +136,8 @@
 }
 
 .add-related-bugs-form {
+  display: flex;
```

Comment:
I apologize, I meant to mention these classes as well for flex:

    d-flex align-items-start

Would you update this PR making use of these classes as well?  Thank you!!

Reply:
whoops. done.